### ISSUE TITLE
dnf [options] remove --oldinstallonly

### ISSUE THREAD

**Issue by j-mracek** [rpm-software-management/dnf5#762]
Removes old installonly packages, keeping only latest versions and version of running kernel.

**Comment by myllynen** [comment]
Now that dnf5 is the default dnf on Fedora 41 there seems to be now easy way to remove e.g. old kernel versions.

Should this issue be prioritized higher to fill the feature gap in Fedora 41 or are there perhaps other plans to address this? Thanks.

**Comment by evan-goode** [comment]
Yes, I think it'd be fair to revisit this. I'll make a note to discuss it with the team. There is already `installonly_limit`. By default it's `3`, but you can set it to `2` to only have 2 kernel versions installed concurrently (running kernel and just-installed kernel).
